src/components/Dialog.js

```diff
@@ -107,26 +108,43 @@ export default class Dialog extends Component {
     window.addEventListener('keydown', this.__onKeyDown, true);
   }
 
+  componentDidMount() {
+    this.closeBtnRef.focus();
```

Comment:
I'm not sure if close button should be focused on the start, it will read out close from the start. what we do in lion is set tabindex -1 on the dialog, focus it, and remove tabindex -1 again

Reply:
see https://github.com/Code-for-All/lockdown/pull/136/commits/50a0182d41f548fca8d793a026bc1559ae0a7f27